src/NetaceaConfig.json

```
@@ -5,5 +5,16 @@
     "kinesisStreamName": "",
     "kinesisAccessKey": "",
     "kinesisSecretKey": ""
-  }
-}
+  },
+  "cookieEncryptionKey": "",
+  "mitigationType": "",
+  "ingestType": "",
+  "netaceaCookieName": "X-Custom-Cookie",
+  "netaceaCaptchaCookieName": "X-Captcha-Cookie",
+  "netaceaCaptchaPath": "/captcha",
+  "captchaHeader": {
+    "name": "X-Captcha-Serve",
+    "value": "true"
+  },
+  "dynamicCaptchaContentType": true
```

Could simplify this, I don't think we need these advanced options as defaults 🙂 
```suggestion
  "cookieEncryptionKey": "",
  "mitigationType": "MITIGATE"
```